### PR TITLE
Added support for Argmax and Argmin operation

### PR DESCRIPTION
**Pull request by nitishat9**
Implemented Argmax and Argmin operation

Signed-off-by: nitishat <nitisha.tomar@intel.com>

### CODE REVIEW

**Review by akodanka — APPROVED** [review]

File: ngraph_creator/operations/src/Argmin.cpp [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    input = getInputNode(0);
+
+    int32_t axis = sModelInfo->ParseOperationInput<int>(mNnapiOperationIndex, 1);
+    ALOGD("createNode axis %d", axis);
+
+    auto k_node = createConstNode(ngraph::element::i32, {}, convertToVector(1));
+
+    auto topk = std::make_shared<ngraph::opset3::TopK>(
+        input, k_node, axis, ngraph::opset3::TopK::Mode::MIN, ngraph::opset3::TopK::SortType::NONE);
```
Comment (RavirajSitaram) [inline ngraph_creator/operations/src/Argmin.cpp:42]: index_element_type can be specified in input itself according to https://docs.openvinotoolkit.org/latest/openvino_docs_ops_sort_TopK_3.html and the topk.hpp in source code.

File: ngraph_creator/operations/src/Argmin.cpp [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    const auto axis_to_remove =
+        createConstNode(ngraph::element::u32, {}, convertToVector(topk->get_axis()));
+    const auto reshaped_indices =
+        std::make_shared<ngraph::opset3::Squeeze>(topk->output(1), axis_to_remove);
+
+    auto outputNode =
```
Comment (RavirajSitaram) [inline ngraph_creator/operations/src/Argmin.cpp:49]: Is Convert OP required if we can specify the element type in constructor?

File: ngraph_creator/operations/src/Argmin.cpp [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    ALOGD("createNode axis %d", axis);
+
+    auto k_node = createConstNode(ngraph::element::i32, {}, convertToVector(1));
+
+    auto topk = std::make_shared<ngraph::opset3::TopK>(
+        input, k_node, axis, ngraph::opset3::TopK::Mode::MIN, ngraph::opset3::TopK::SortType::NONE);
+
+    const auto axis_to_remove =
+        createConstNode(ngraph::element::u32, {}, convertToVector(topk->get_axis()));
+    const auto reshaped_indices =
```
Comment (RavirajSitaram) [inline ngraph_creator/operations/src/Argmin.cpp:46]: NNAPI requires the ouput dimensions to be (N-1). Squeeze may impact the final dimensions ? Is there a specific reason for Squeeze operation here?

File: ngraph_creator/operations/src/Argmin.cpp (hunk 2 above)
Comment (akodanka) [reply ngraph_creator/operations/src/Argmin.cpp:49]: Comment valid for Line#47.
The NNAPI documentation specifies output to be "An (n - 1)-D Tensor".
The value for k that we pass to ngraph TopK is 1, resulting in dimension 1 along the specified axis. So, Squeeze with that axis would remove the dim 1 at the axis, resulting in the desired (n-1)-D tensor.

File: ngraph_creator/operations/src/Argmin.cpp (hunk 3 above)
Comment (nitishat9) [reply ngraph_creator/operations/src/Argmin.cpp]: Taken care of in the latest commit

**Review by RavirajSitaram — COMMENTED** [review]
LGTM

**Review by JeevakaPrabu — APPROVED** [review]

**Pull request merged**